database/schema.sql

```diff
@@ -0,0 +1,181 @@
+
+-- Valid for squlite3 3.10+ (if not exists, without rowid).
+
+-- As sql keywords are composite and have spaces inside them,
+-- I used 2 spaces to separate names, types and constraints visually.
+--
+-- Also, sqlite is said to create indices on PKs automatically, so I will omit them.
+--
+-- As the sqlite will create additional `rowid` unique PK even if another
+--  non integer or integer desc PK is present, some of tables are created
+--  WITHOUT ROWID.
+
+-- Creating 'Courses' table.
+--
+create table if not exists Courses (
+
+    -- Sqlite3 will force ascending primary key to be non-null, replacing
+    --  null with autoincremented key (while sqlite2 won't).
+    --
+    id    INTEGER,
+    desc  TEXT     null,
+
+    primary key (id asc)
+);
+
+-- Creating 'Subjects' table.
+--
+create table if not exists Subjects (
+    id         INTEGER,
+    course_id  INTEGER  not null,
+
+    primary key (id asc),
```

Comment:
1) Not `asc`, subject IDs are part of ATG and should be the same for the whole network.
2) I made a mistake in PDM, in `Subjects` table there should be composite primary key `(subject_id, course_id)`, because one subject can be related to many courses. I'll fix PDM and add update to it to this PR.

Reply:
Will fix.